Cargo.toml

```diff
@@ -13,7 +13,7 @@ exclude = ["/core"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-leptos = "0.5.1"
+leptos = "0.5"
```

Comment:
Let me know if you prefer the more specific 3 digit version number.

Reply:
This is fine